fighters/buddy/src/acmd/aerials.rs

```diff
@@ -1,46 +1,114 @@
     if is_excute(fighter) {
-        WorkModule::off_flag(boma, *FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING);
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 9.0, 45, kbg,0,bkb, size, 0.0, 13.0, -14.5, None, None, None, 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_KICK, *ATTACK_REGION_PUNCH);
+    }
+    //Back hitbox lasts for one more frame, the bodybox lasts for an additional frame
+    wait(lua_state, 1.0);
+    if is_excute(fighter) {
+        AttackModule::clear(fighter.module_accessor, 0, false);
+    }
+    wait(lua_state, 1.0);
+    if is_excute(fighter) {
+        AttackModule::clear_all(fighter.module_accessor);
+    }
+    //Pop hitbox on frame 25, adds an extra frame of hitstun on the sweetspot
+    frame(lua_state, 25.0);
+    if is_excute(fighter) {
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 5.0, 92, 32, 0, 77, size-0.25, 0.0, 7.5, 15.0, None, None, None, 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_M, *COLLISION_SOUND_ATTR_KICK, *ATTACK_REGION_PUNCH);
+
+        ATTACK(fighter, 1, 0, Hash40::new("top"), 3.0, 45, kbg,0,bkb+5, size-0.25, 0.0, 8.0, 3.0, Some(0.0), Some(7.0), Some(8.0), 1.01, 0.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_F, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_KICK, *ATTACK_REGION_PUNCH);
+
+        AttackModule::set_add_reaction_frame_revised(boma, 0, 1.0, false);
+    }
+    wait(lua_state, 3.0);
+    if is_excute(fighter) {
+        AttackModule::clear_all(fighter.module_accessor);
+    }
+    frame(lua_state, 42.0);
+    if is_excute(fighter) {
+        WorkModule::off_flag(fighter.module_accessor, *FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING);
+    }
+}
+#[acmd_script( agent = "buddy", script = "effect_attackairn", category = ACMD_EFFECT )]
+unsafe fn buddy_attack_air_n_effect(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    frame(lua_state, 5.0);
+    if is_excute(fighter) {
+        LANDING_EFFECT_FLIP(fighter, Hash40::new("sys_spin_wind"), Hash40::new("sys_spin_wind"), Hash40::new("shoulderr"), 0, -2, 0, 0, 0, 0, 2.5, 0, 0, 0, 0, 0, 0, false, *EF_FLIP_NONE);
+    }
+    frame(lua_state, 17.0);
+    if is_excute(fighter) {
+        LANDING_EFFECT_FLIP(fighter, Hash40::new("sys_spin_wind"), Hash40::new("sys_spin_wind"), Hash40::new("shoulderr"), 0, -2, 0, 0, 0, 0, 2.5, 0, 0, 0, 0, 0, 0, false, *EF_FLIP_NONE);
+    }
+}
```

Comment:
```suggestion
#[acmd_script( agent = "buddy", script = "effect_attackairn", category = ACMD_EFFECT )]
unsafe fn buddy_attack_air_n_effect(fighter: &mut L2CAgentBase) {
    let lua_state = fighter.lua_state_agent;
    frame(lua_state, 9.0);
    if is_excute(fighter) {
        EFFECT_FOLLOW_FLIP(fighter, Hash40::new("sys_spin_wind"), Hash40::new("sys_spin_wind"), Hash40::new("rot"), 0, 2.5, 0, 0, 180, 0, 1.5, true, *EF_FLIP_YZ);
        LAST_EFFECT_SET_RATE(fighter, 0.6);
    }
}
```
The current effect for NAir feels awkward, since it leaves behind two wind trails, which don't follow the actual trajectory of the backpack. Here's my suggestion, using the same effect but with `EFFECT_FOLLOW_FLIP` and only one effect with a rate modifier. Here is the effect after my modification:

https://user-images.githubusercontent.com/33226440/204070251-2fd17c9f-fd6c-4f8a-8019-2c2bfbcd5f10.mp4

Reply:
Ya this does feel much more tight with the animation